feat:Create Ping Section

AianMarkAntonio opened the pull request:
Added css on header,  edited ping.php and customized/added ping.css

AianMarkAntonio committed:
feat:Create Ping Section

Added css on header,  edited ping.php and customized/added ping.css

Christiann12 reviewed (changes requested):
Change ping definition.

AianMarkAntonio committed:
feat: Created modal

Added modal, modified the steps description in pinghow container and added some css for modals

Christiann12 reviewed (approved):
Work's perfectly

Christiann12 committed:
Merge branch 'main' into issue-61
